lib/gpu_memory_service/core/server/rpc.py

```
@@ -0,0 +1,127 @@
+                    request = self._receive()
+                except EOFError:
+                    return
+                export_fd = -1
+                try:
+                    if not isinstance(request, REQUEST_TYPES):
+                        raise RuntimeError(
+                            "handshake is valid only as the first message"
+                        )
+                    response, export_fd = manager.handle_request(session, request)
+                    send_message(self.request, response, export_fd)
+                except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)